src/lib.rs

```diff
@@ -261,6 +303,15 @@ impl Builder {
         Ok((logger, bg_task))
     }
 
+    /// Return `Logger` and TCP connection background task.
+    fn connect_tcp<T>(self, addr: T) -> Result<(Logger, BackgroundTask), BuilderError>
```

Comment:
```suggestion
    pub fn connect_tcp<T>(self, addr: T) -> Result<(Logger, BackgroundTask), BuilderError>
```

Looks like this was accidentally made non-pub.